src/components/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,35 @@
+import { NavLink } from "react-router-dom";
+import IconAsSvg, { SvgDetails } from "../IconAsSvg/IconAsSvg";
+
+interface Props {
+  buttonDestination: string;
+  buttonText: string;
+  withArrowIcon?: boolean;
+  arrowIconDetails?: SvgDetails;
+  className: string;
+  onClick?: () => void;
+}
+
+const ButtonLink = (props: Props): JSX.Element => {
+  const {
+    buttonDestination,
+    buttonText,
+    withArrowIcon,
```

Comment:
Awesome!  What should arrowIconDetails be by default?  It must match the type of SvgDetails but it's an optional argument.  Or is it not necessary to provide a default value here because it works in conjunction with the Boolean withArrowIcon which is false by default?  Thanks!!

Reply:
Oh, also, inside ProductInfo component, there is an optional prop price.  This is because on the CategoryPage we don't want to display a price, but on ProductPage we do (both of those pages are using this component to display information about the products).  If I give a default value of 0 to price, I'm seeing a 0 appear on the CategoryPage but I don't want that.  I want nothing to render unless a price is given.  Is there a default value I can assign a number that will equate to false?  In node, 0 seems to evaluate as falsy, but on the actual app I'm seeing a p tag appear with 0 on it.  We ain't givin' this stuff away for free!